### PR TITLE
Add cloudbuild tag config

### PR DESCRIPTION
**Pull request by gibbleyg**
/gcbrun

### CODE REVIEW

File: cloudbuild-tag.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+- id: Pull Staging Image
+  name: gcr.io/cloud-builders/docker
+  args: ["pull", "gcr.io/$_STAGING_PROJECT_ID/mpdev:sha_$COMMIT_SHA"]
+- id: Tag Image
+  name: gcr.io/cloud-builders/docker
+  args: ["tag", "gcr.io/$_STAGING_PROJECT_ID/mpdev:sha_$COMMIT_SHA",
+      "gcr.io/$PROJECT_ID/mpdev:$TAG_NAME"]
+
+images:
+- gcr.io/$PROJECT_ID/mpdev:$TAG_NAME
```
Comment (gibbleyg) [inline cloudbuild-tag.yaml:14]: @huyhg not sure if we want to add an extra directory in this image name. I noticed that some of the other marketplace-tools images are gcr.io/PROJECT_ID/k8s/IMAGE_NAME etc.

Comment (huyhg) [reply cloudbuild-tag.yaml:14]: `mpdev` can sit at the top level since it's more foundational compared to other container images here.

**Review by huyhg — APPROVED** [review]

**Pull request merged**